python/Ganga/Lib/Batch/Batch.py

```diff
@@ -426,6 +426,7 @@ def preparejob(self, jobconfig, master_input_sandbox):
         #'###SHAREDINPUTPATH###' : repr(sharedinputpath)),
 
         '###INPUT_SANDBOX###' : repr(subjob_input_sandbox + master_input_sandbox),
+            '###CREATEINPUTDATALIST###' : getWNCodeForInputdataListCreation(job, ''),
```

Comment:
minor point, this indentation doesn't line up but as it's an element of a `dict` it should be fine so feel free to line it up or not but nothing that stops a merge